iset.mm

```diff
@@ -154430,6 +154468,74 @@ equivalent way to state real trichotomy (see further discussion at
       $.
   $}
 
+  ${
+    $d A f g x $.  $d G f g x $.  $d G j $.  $d V f g x $.
+    ismkvnnlem.g $e |- G = frec ( ( x e. ZZ |-> ( x + 1 ) ) , 0 ) $.
+    $( Lemma for ~ ismkvnn .  The result, with a hypothesis to give a name to
+       an expression for convenience.  (Contributed by Jim Kingdon,
+       25-Jun-2024.) $)
+    ismkvnnlem $p |- ( A e. V -> ( A e. Markov <-> A. f e. ( { 0 , 1 } ^m A )
```

Comment:
Maybe a lemma could be extracted from this (long) proof to show the relevance/some properties of the class `G`.

Reply:
I definitely thought about it. We have some properties of the mapping ` G ` at:

* https://us.metamath.org/ileuni/frec2uz0d.html through https://us.metamath.org/ileuni/frec2uzisod.html
* https://us.metamath.org/ileuni/frechashgf1o.html
* https://us.metamath.org/ileuni/frec2uzled.html

and various others. Maybe a few candidates would be:

* (𝐺 ↾ 2o):2o⟶{0, 1}
* (◡𝐺 ↾ {0, 1}):{0, 1}⟶2o

The goal, of course, is to have some lemmas which we can also use in https://us.metamath.org/ileuni/isomninnlem.html and https://us.metamath.org/ileuni/iswomninnlem.html .

I'll take a look at revising this pull request to add those two. At least as far as I noticed glancing over the proofs, those seemed to be the easiest lemmas to split out.

Reply:
These lemmas are now added and are both used in `isomninnlem`, `iswomninnlem`, and `ismkvnnlem`.